docs/install/airgapped-clusters.md

```diff
@@ -0,0 +1,365 @@
+                operator: In
+                values:
+                - storageos-etcd
+            topologyKey: kubernetes.io/hostname
+          weight: 100
+    etcdEnv:
+    - name: ETCD_HEARTBEAT_INTERVAL
+      value: "500"
+    - name: ETCD_ELECTION_TIMEOUT
+      value: "5000"
+    - name: ETCD_MAX_SNAPSHOTS
+      value: "10"
+    - name: ETCD_MAX_WALS
+      value: "10"
+    - name: ETCD_QUOTA_BACKEND_BYTES
+      value: "8589934592"
+    - name: ETCD_SNAPSHOT_COUNT
+      value: "100000"
+    - name: ETCD_AUTO_COMPACTION_RETENTION
+      value: "20000"
+    - name: ETCD_AUTO_COMPACTION_MODE
+      value: revision
+    resources:
+      limits:
+        cpu: 200m
+        memory: 200Mi
+      requests:
+        cpu: 200m
+        memory: 200Mi
+  replicas: 3
+  storage:
+    volumeClaimTemplate:
+      resources:
+        requests:
+          storage: 128Gi
+      storageClassName: local-path
+  tls:
+    enabled: false
+    storageOSClusterNamespace: storageos
+    storageOSEtcdSecretName: storageos-etcd-secret
+```
+
+
+## Step 5. Installing the cluster
+
+Once all the YAMLs are ready, the Ondat cluster can be bootstrapped. 
+
+### Option 1: With Etcd in Kubernetes
+
+The etcd cluster in Kubernetes requires a StorageClass. If you are running on
+a cloud provider, you can use existing StorageClasses for it, for example `gp3` (AWS) or
+`standard` (GCE). Or you can create the `local-path` StorageClass. It's
+recommended to give Etcd a backend disk that can sustain at least 800 IOPS. It's best 
+to use provisioned IOPS when possible, otherwise make sure the size of
+the disk is big enough to fulfil the IOPS requirement when performance depends
+on IOPS per GB.  For example, AWS would require a burstable EBS bigger than
+256G to fulfil the IOPS requirement.  
+
+⚠️  The `local-path` StorageClass is only recommended for __non production__
+clusters as the data of the etcd peers is susceptible of being lost on node
+failure.
+
+```bash
+# Local-path StorageClass
+# Not for production workloads
+kubectl apply -f https://raw.githubusercontent.com/rancher/local-path-provisioner/master/deploy/local-path-storage.yaml
+```
+
+1. Edit the variable `ETCD_STORAGECLASS`
+
+```bash
+ETCD_STORAGECLASS=my-storage-class
+# Edit the StorageClass for etcd
+sed -i -e "s/storageClassName: local-path/storageClassName: $ETCD_STORAGECLASS/g" etcd-cluster.yaml
+```
+
+
+2. Edit the variable `ETCD_STORAGECLASS`
+```
+# Install (the kubectl-storageos plugin installs Etcd and Ondat)
+ETCD_STORAGECLASS=my-storage-class
+ONDAT_VERSION=v2.5.0
+kubectl storageos install \
+	--include-etcd \
+	--etcd-storage-class $ETCD_STORAGECLASS \
+	--etcd-namespace storageos \
+	--etcd-operator-yaml etcd-operator.yaml \
+	--etcd-cluster-yaml etcd-cluster.yaml \
+  	--skip-etcd-endpoints-validation \
+	--stos-operator-yaml ondat-operator.yaml \
+	--stos-cluster-yaml ondat-cluster.yaml \
+	--stos-cluster-namespace storageos \
+	--stos-version $ONDAT_VERSION
```

Comment:
In my experience you also needed the `--resource-quota-yaml` parameter here as well. Is that not needed anymore @nolancon?

Reply:
I didn't needed to get the cluster running, but I'd like to know that the --resource-quota-yaml is, actually?